src/Utils/track.ts

```diff
@@ -95,7 +95,9 @@ export function trackWithProps<P, T extends TrackingInfo.Global = TrackingInfo.G
  *        }
  *      }
  */
-export function trackWithoutProps<T extends TrackingInfo.Global = TrackingInfo.Global>(): Track<T, any> {
+export function trackWithoutProps<
+  T extends Global = Global
+>(): Track<T, any> {
   return _track
 }
 
```

Comment:
/cc @mennenia - we may eventually need to make similar changes (removing the `namespace`) in Emission - [context here in slack](https://artsy.slack.com/archives/C2TQ4PT8R/p1519255117000112)